charts/weblate/values.yaml

```diff
@@ -96,6 +96,9 @@ ingress:
 persistence:
   enabled: true
 
+  # existingClaim -- Use an existing volume claim
```

Comment:
try simply with 

```
# -- Use an existing volume claim
```

Reply:
thanks for the hint! but somehow the check is still not happy.

Reply:
got it. at least the readme part